provision/provision.sh

```diff
@@ -57,19 +57,17 @@ export VVV_PACKAGE_REMOVAL_LIST=()
 . "/srv/provision/core/nginx/provision.sh"
 . "/srv/provision/core/memcached/provision.sh"
 . "/srv/provision/core/php/provision.sh"
-. "/srv/provision/core/composer/provision.sh"
-. "/srv/provision/core/nodejs/provision.sh"
-. "/srv/provision/core/grunt/provision.sh"
 . "/srv/provision/core/mailhog/provision.sh"
-. "/srv/provision/core/wp-cli/provision.sh"
-. "/srv/provision/core/phpcs/provision.sh"
+. "/srv/provision/core/nodejs/provision.sh"
```

Comment:
this isn't a tool after all? should be part of that provisioner?

Reply:
Which? PHPCS and WP CLI are tools so they're moved to the tools provisioner, Node is a base dependency

Reply:
it was node but ok

Reply:
Ah, Node isn't a tool